### DISCUSSION TITLE
Have a 'after' or 'triggers' option for a pipeline? Use eg for outputting all dev urls at the end of dev command.

### DISCUSSION THREAD

**Discussion by JeroenReumkens** [vercel/turborepo#540] [Ideas]
Hey @jaredpalmer,

I'm thinking whether a `after` or `triggers` option for a pipeline would be helpful. In this specific case I want to output all different dev urls after `turbo run dev` is ran. Currently those URLs are lost because different modules output URLs at different times, with other logs in between. 

If we add a `after` function that could trigger a command, we could run a specific task (part of the project code, not part of turbo) that outputs these to the console. Another option would be to have some specific functionality baked into Turbo to output urls of all apps, but I think a `after` property could be used for different purposes in the future as well (less opinionated). 

Would love to hear what you think about something like this!

**Comment by anthonyshew** [comment]
This wouldn't quite work since the processes for dev commands are long-running, meaning that they don't end once your dev server is done spinning up. Not necessarily a limitation of `turbo`...but of computers!
